vault/asset.preview.md

```diff
@@ -0,0 +1,43 @@
+---
+id: ChcCM1Q2qrDTy7YnLUHiL
+title: Preview
+desc: ''
+updated: 1639199963864
+created: 1639113925844
+---
+
+> This is a collection of assets used to insert into documentation via note refs.
+
+## Tutorial
+
+### Tutorial Welcome Screen
+
+![Tutorial Welcome Screen](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-welcome-screen-2.png)
+
+### Tutorial Launch Command
```

Comment:
we mix themes here - with light theme vs dark theme. can we consolidate on one:? and update screenshots?

lets keep ligh theme assets around for future use when we want to show both themes but for now lets just do dark theme

for assets, for each asset, we can create a subheading (dark/light)

Reply:
lets have a definite theme for screenshots. the default dark theme for vscode is "Default Dark+". I personally like github's dark theme, https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme

Reply:
On the GitHub Dark theme: Noticed some things
- `GitHub Dark` in that extension is considered legacy, and left for the purposes that others can still use it but it may not be updated. Alternative and maintained themes included in that extension are: `GitHub Dark Default` or `GitHub Dark Dimmed`. I'm going with `GitHub Dark Default`. :sunglasses: 
- There seems to be a bug where all the GitHub themes don't change highlighting for `inline code like this`, causing that text to just look like regular text. There has been a PR open for this since August, it looks like, but hasn't had much for traction. I'm trying to see if I can apply the fix locally. In the meantime, this shouldn't be an issue for what most of our screenshots since we are highlighting other features. PR I'm referencing: https://github.com/primer/github-vscode-theme/pull/193

Reply:
> On the GitHub Dark theme: Noticed some things
> 
>     * `GitHub Dark` in that extension is considered legacy, and left for the purposes that others can still use it but it may not be updated. Alternative and maintained themes included in that extension are: `GitHub Dark Default` or `GitHub Dark Dimmed`. I'm going with `GitHub Dark Default`. sunglasses
> 
>     * There seems to be a bug where all the GitHub themes don't change highlighting for `inline code like this`, causing that text to just look like regular text. There has been a PR open for this since August, it looks like, but hasn't had much for traction. I'm trying to see if I can apply the fix locally. In the meantime, this shouldn't be an issue for what most of our screenshots since we are highlighting other features. PR I'm referencing: [Update markdown inline code scope primer/github-vscode-theme#193](https://github.com/primer/github-vscode-theme/pull/193)

With the above in mind, we are going with `Dark+` to make things easy.